cli/cmd/plugin/ui/web/tanzu-ui/src/components/DockerForm.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ChangeEvent, useContext } from "react";
+
+import { CdsInput } from "@cds/react/input";
+import { CdsFormGroup } from "@cds/react/forms";
+import { CdsButton } from "@cds/react/button";
+
+import { TEXT_CHANGE } from "../constants/actionTypes";
+import { Store } from "../stores/store";
+
```

Comment:
Please remove extra line break. Linting rules that I'm adding should catch this in the future.

Reply:
Let's wait for your linter PR merged. Then I will run it and check whether I violate any other rules

Reply:
Okay sounds good. I'll get it merged asap.